applications/nrf_desktop/src/events/Kconfig

```diff
@@ -11,72 +11,104 @@ menu "Event options"
 comment "Initially logged events"
 
 config DESKTOP_INIT_LOG_BATTERY_STATE_EVENT
-	bool "Battery state event"
+	bool "Log battery state events"
 	default y
+	help
+	  Log battery state events in nRF Desktop application.
 
 config DESKTOP_INIT_LOG_BATTERY_LEVEL_EVENT
-	bool "Battery level event"
+	bool "Log battery level events"
 	default y
+	help
+	  Log battery level events in nRF Desktop application.
 
 config DESKTOP_INIT_LOG_BLE_DISC_COMPLETE_EVENT
-	bool "BLE discovery complete event"
+	bool "Log Bluetooth LE discovery complete events"
 	default y
```

Comment:
Help is missing

Reply:
Must have remove it by mistake, thanks - will add it.